### ISSUE TITLE
igxSelectComponent accessibility issue

### ISSUE THREAD

**Issue by teodosiah** [IgniteUI/igniteui-angular#8079]
## Description  
Dropdown menus (select boxes or role="listbox") should always have a description that is explicitly associated with the field to make sure that users of assistive technologies will also know what the field is for.
If the drop-down menu (select box or role="listbox") has a visible description indicating the purpose of the menu, this description should be explicitly associated with the input drop-down menu either as an HTML label or using the WAI-ARIA attribute 'aria-labelledby'.
If it is not possible to add a visible description, either add a mouseover text ('title' attribute) to the drop-down menu or create an invisible label using the WAI-ARIA attribute 'aria-label'.

 * igniteui-angular version: 10.1.2

## Steps to reproduce  

1. Run the sample
2. Open the narrator and navigate through it.

## Result  
No description on the drop-down.

## Expected result  
Dropdown menus (select boxes or role="listbox") should always have a description that is explicitly associated with the field to make sure that users of assistive technologies will also know what the field is for.

## Attachments  
Sample application could be found [here](https://stackblitz.com/edit/igx-select-aria?file=src/app/select-sample-1/select-sample-1.component.ts)

**Comment by IvayloG** [comment]
We could additionally provide an _**aria-label**_ attribute for the **_listbox_** in case there is no label element specified(so **_aria-labelledby_** can't be used). On the other hand, we may consider that if there is no label element explicitly set it is a developer’s a choice and should not force a default _**aria-label**_  attribute.
